src/frontend/src/pages/CreateChangeRequestPage/CreateProposedSolutionsList.tsx

```diff
@@ -37,7 +36,28 @@ const CreateProposedSolutionsList: React.FC<CreateProposedSolutionsListProps> =
 
   return (
```

Comment:
change the auth = useAuth call to be user = useCurrentUser (it's smthn we added, but hasn't been updated everywhere)